language-design.md

```diff
@@ -106,7 +162,23 @@ you do so? If not, why not?
 
 **Response**
 
-
+Not necessarily; Quorum is more structured than AppleScript, so it's possible to include
+natural language, and thus make the language more understandable to novice users, as long
+as the language is structured enough for the user to parse it as proper code. Where AppleScript
+fails is in making the syntax too similar to English, but the actual parsing doesn't support
+syntax that would be acceptable in English, and may be written by the users,
+but isn't handled by the language.
+Quorum is structured enough for its users to understand that their code must be structured,
+so while it's easier for them to understand there is less of a chance they will make syntax errors.
+
+Whether or not to use natural language depends on many factors, such as the purpose of the language and who its users will be, and 
+what parsing libraries are available for natural language syntax. For example, 
```

Comment:
Parsers are pretty computationally expensive, too. Though, to be fair, there are certainly optimizations that you can make to make it less intensive, as seen [here](http://www.elliotglaysher.org/data/glaysher06.pdf)